en/publish/publish-bytedance.md

```diff
@@ -0,0 +1,124 @@
+
+  ![](./publish-bytedancegame/appid.png)
+
+## Publishing Process
+
+Use Cocos Creator to open the project to be published, open the **Build...** panel from **Menu Bar -> Project**, select **ByteDance Mini Game** in **Platform**, fill in the appid, and click **Build**.
+
+![](./publish-bytedancegame/build.png)
+
+### Parameter Configuration
+
+The specific rules for filling in the relevant parameter configuration are as follows.
+
+- **The Compression Type of the Main Package**
+
+  To set the compression type of the main package, please refer to the [Asset Bundle - Compression Type](../asset-manager/bundle.md#compression-type) documentation.
+
+- **Configure the Main Package as a Remote Package**
+
+  This is optional and needs to be used in conjunction with the **Resource Server Address** option. <br>
+  When checked, the main package is configured as a remote package and is built with its associated dependencies to the built-in Asset Bundle -– [main](../asset-manager/bundle.md#the-built-in-asset-bundle) in the remote directory of the release package. The developer needs to upload the entire remote folder to the remote server.
+
+- **Initial Scene Subpackage**
+
+  This item is optional. <br>
+  When checked, the first scene and its related dependencies will be built into the built-in Asset Bundle under the release package directory assets –– [start-scene](../asset-manager/bundle.md#the-built-in-asset-bundle) to improve the speed of loading assets for the initial scene. The details can be found in the document [Asset Loading for Initial Scenes](publish-wechatgame.md#speed-up-the-loading-of-the-start-scene).
+
+- **Resource Server Address**
+
+  This item is optional and is used to fill in the address of the remote server where the resource is stored. Developers need to manually upload the remote folder in the release package directory to the filled resource server address after the build.
+
+- **Code Directory for Open Data Domains**
+
+  This item is optional and is used to access the open data domain of ByteDance , please refer to [Access the Open Data Domain of ByteDance Mini Game](publish-bytedance-sub-domain.md) for the specific publishing process.
+
+### Run Preview
+
+- Click the **Open** button behind **Build Path** after the build is finished, the ByteDance mini game project folder **bytedance** is generated under the build directory of the publish package, which already contains the configuration files of bytedance  environment: `game.json` and `project. config.json`.
+
+  ![](./publish-bytedancegame/package.png)
+
+- Use the **ByteDance developer tool** to open the **bytedance** folder generated by the build, open the ByteDance game project and preview the debugging game content. Please refer to [Introduction to ByteDance Developer Tools](https://microapp.bytedance.com/docs/zh-CN/mini-game/develop/developer-instrument/development-) for details on how to use the developer tools. assistance/mini-app-developer-instrument).
```

Comment:
Something is wrong, looks like the url was split with text?

[Introduction to ByteDance Developer Tools](https://microapp.bytedance.com/docs/zh-CN/mini-game/develop/developer-instrument/development-) for details on how to use the developer tools. assistance/mini-app-developer-instrument).

Reply:
Please refer to [Introduction
-->
Please refer to the [Introduction